forms/viewsets/form.py

```diff
@@ -109,6 +117,49 @@ def get_serializer_class(self):
         return super().get_serializer_class()
 
 
+class AnswerPublicViewSet(AnswerViewSet):
```

Comment:
I would maybe inherit this from mixins.CreateModelMixin and viewsets.GenericViewSet. If there is something needed from AnswerViewSet, I would create AnswerMixin

Reply:
Probably nothing, so your suggestion is great and I will do it like that